update-repo.py

```diff
@@ -0,0 +1,140 @@
+import subprocess
+import sys
+import os
+
+import shutil
+
+
+"""
+This script streamlines the process of copying a .yml file into each workflows directory, 
+followed by creating a PR into the GitHub feedstock repository.
+
+How to use:
+
+python /path/.../update-repo.py <filepath-you-want-to-copy-in-your-local-machine> <filepath-in-the-github-repo>
```

Comment:
could you please also add an example for the case of `.yml`? that would be really helpful for new users.